tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -16,8 +16,12 @@ setup() {
     [ "${SNAPSHOTTER:-}" = "nydus" ] || skip "None snapshotter was found but this test requires one"
 
     setup_common
+    get_pod_config_dir
     unencrypted_image="quay.io/prometheus/busybox:latest"
-    large_image="quay.io/confidential-containers/test-images:largeimage"
+    image_pulled_time_less_than_default_time="rust:latest" # unpacked size: 1.41GB
```

Comment:
`rust:latest` will be pulled from docker.io so changes are CI will hit pull rate limit issue. We better find a large image from quay.io, or maybe this image is not needed (see my next comment)

Reply:
Yes, I agree. But I’ve been searching for an appropriate and dependable image from quay.io that is larger than 1GB yet smaller than 2GB, but I haven’t found one that meets these criteria yet. Do you have any advice?

Reply:
@wainersm  I have replaced "rust:latest" with "ghcr.io/confidential-containers/test-container:rust-1.79.0". Thanks @Xynnn007's help!